contracts/script/avs/DeployGoerliAVS.s.sol

```diff
@@ -52,13 +50,25 @@ contract DeployGoerliAVS is Script {
                 owner,
                 portal,
                 omniChainId,
-                minimumOperatorStake,
-                maxOperatorCount,
+                MIN_OPERATOR_STAKE,
+                MAX_OPERATOR_COUNT,
                 allowlist,
                 StrategyParams.goerli()
             )
         );
 
         return proxy;
     }
+
+    /// @dev deploy OmniAVS, but with a prank. necessary because we cannot
+    //       vm.startPrank() outside of this contract
```

Comment:
I thought if `vm.startPrank` is called, all txns are called with that address until stopped – regardless of where the calls happen

Reply:
I did too so I wrote it that way first. But it didn't work. It's just all calls from the contract in which `vm.startPrank` was called. But that actually makes sense. You don't want `vm.startPrank()` to affect all `msg.sender`s in a chain of function calls

So

```
contract A {
     function foo() {
        vm.startPrank(me);
        b.foo();
    }
}

contract B {
    function bar() {
        // msg.sender == me
        c.baz()
   }
}


contract C {
   function baz() {
       // msg.sender == address(b)
   }
}
```